turborepo-tests/integration/tests/run-summary/error.t

```diff
@@ -80,7 +80,7 @@ Validate that we got a full task summary for the failed task with an error in .e
     "execution": {
       "startTime": [0-9]+, (re)
       "endTime": [0-9]+, (re)
-      "error": "command .* npm run maybefails exited \(1\)", (re)
+      "error": "command (.*)npm run maybefails exited \(1\)", (re)
```

Comment:
This came from Rust tests, I thiink it's unrelated to this PR, but I was cross cehcking that test suite to make sure anything didn't break